Cork/Localizable.xcstrings

```diff
@@ -14434,6 +14578,12 @@
             "state" : "needs_review",
             "value" : "error.package-loading.last-path-component-of-checked-package-url-is-folder-containing-packages-itself.formulae"
           }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "所提供的路径指向 Formulae 文件夹本身，并非指向软件包。 你的 Homebrew 程序损坏，这并非是Cork的问题"
```

Comment:
We just need to keep the singular form of Formula/Cask/Tap in Chinese translations since we don't have the concept of plurals anyway